lib/ansible/plugins/doc_fragments/panos.py

```diff
@@ -25,3 +25,221 @@ class ModuleDocFragment(object):
         type: str
         default: admin
 '''
+
+    PROVIDER = r'''
+options:
+    provider:
+        description:
+            - A dict object containing connection details.
+        version_added: 2.8
+        required: true
+        suboptions:
+            ip_address:
+                description:
+                    - The IP address or hostname of the PAN-OS device being configured.
+                type: str
+                required: true
+            username:
+                description:
+                    - The username to use for authentication.  This is ignored if
+                      I(api_key) is specified.
+                type: str
+                default: 'admin'
+            password:
+                description:
+                    - The password to use for authentication.  This is ignored if
+                      I(api_key) is specified.
+                type: str
+            api_key:
+                description:
+                    - The API key to use instead of generating it using
+                      I(username) / I(password).
+                type: str
+            port:
+                description:
+                    - The port number to connect to the PAN-OS device on.
+                type: int
+                default: 443
+            serial_number:
+                description:
+                    - The serial number of a firewall to use for targeted commands.
+                      If I(ip_address) is not a Panorama PAN-OS device, then
+                      this param is ignored.
+                type: str
+'''
+
+    TRANSITIONAL_PROVIDER = r'''
+options:
+    provider:
+        description:
+            - A dict object containing connection details.
+        version_added: 2.8
```

Comment:
```suggestion
        version_added: '2.8'
```

Reply:
Resolved in 0595acc4f78f55103e406b523b7fd67115105162